### ISSUE TITLE
Proper Documentation

### ISSUE THREAD

**Issue by thubamamba** [aws-samples/textract-bank-statement-processor#1]
Stumbled upon this repo... would be very helpful if not only was the process documented (which it is), but also setting this repo up and getting it up and running.

**Comment by kostabasis** [comment]
Agreed, this tool is amazing but it's very hard to figure out how to use it locally as a package in python. Please leave any example setups in this issue for all who see this. Documentation would be super helpful.
